src/internal/m365/service/sharepoint/backup.go

```diff
@@ -80,7 +80,7 @@ func ProduceBackupCollections(
 			spcs, canUsePreviousBackup, err = site.CollectLibraries(
 				ctx,
 				bpc,
-				drive.NewLibraryBackupHandler(ac.Drives(), scope),
+				drive.NewLibraryBackupHandler(ac.Drives(), scope, path.SharePointService),
```

Comment:
Nit:  and could probably do this other places, too.
```suggestion
				drive.NewLibraryBackupHandler(ac.Drives(), scope, bpc.Selector.PathService()),
```